types/moji/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for moji 0.5
+// Project: https://github.com/niwaringo/moji
+// Definitions by: Yasunori Ohoka <https://github.com/yasupeke>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+declare namespace moji {
+  type StringType = 'ZE' | 'HE' | 'ZS' | 'HS' | 'HG' | 'KK' | 'ZK' | 'HK';
+
+  interface MojisyuRange {
+    start: number;
+    end: number;
+  }
+
+  interface MojisyuRegexp {
+    regexp: RegExp;
+    list: string[];
+  }
+
+  class Moji {
```

Comment:
While there is a `class` declared in the library, it isn't exposed on the `moji` object itself. Consider converting this to an `interface` and removing the `constructor`.